CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased Changes
+
+## Enhancements
+* Remove beta messaging and cleanup for `RunListOptions` struct by @mjyocca [#472](https://github.com/hashicorp/go-tfe/pull/472)
```

Comment:
It's not a BREAKING CHANGE, because we warned that it might change, but it would be good to call out the field rename in RunListOptions (Name -> User)